docs/dev/table/hive/read_write_hive.md

```diff
@@ -120,4 +120,11 @@ Flink SQL> INSERT INTO mytable (name, value) VALUES ('Tom', 4.72);
 
 ### Limitations
 
-Currently Flink's Hive data connector does not support writing into partitions. This feature is under active development.
+The following is a list of major limitations of the Hive connector. And we're actively working to close these gaps.
+
+1. INSERT OVERWRITE is not supported.
+2. Inserting into partitioned tables is not supported.
+3. ACID tables are not supported.
+4. Bucketed tables are not supported.
+5. Some data types are not supported. See the [limitations]({{ site.baseurl }}/dev/table/hive/#limitations) for details.
+6. Only a limited number of table storage formats have been tested, namely text, ORC, and Parquet.
```

Comment:
do we need extra dependencies if users use ORC/Parquet?

A user trying to use lzo found that he needed to include hadoop-lzo himself
http://apache-flink.147419.n8.nabble.com/flink1-10-hive-td336.html

Reply:
I don't think so according to our previous tests. Also manually verified reading ORC/Parquet tables with different compression codec and seems we're good.